packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -317,9 +320,9 @@ class AndroidCameraCameraX extends CameraPlatform {
   @override
   Future<void> dispose(int cameraId) async {
     preview?.releaseFlutterSurfaceTexture();
-    liveCameraState?.removeObservers();
+    await liveCameraState?.removeObservers();
     processCameraProvider?.unbindAll();
-    imageAnalysis?.clearAnalyzer();
+    await imageAnalysis?.clearAnalyzer();
```

Comment:
I think I fixed the unawaited issues recently. Can you merge in the latest changes?

Reply:
Merged.
But I previously added `await` stuff instead of just 'unawaited' in analysys conflicts cases and android builds becomes more stable. With just `unawaited` there was infrequent runtime race condition errors between java and dart. Like `unable to close already closed camera resource` or `unable to close: camera resource is used`.

Reply:
Gotcha. I'm certainly open to reviewing any `unawaited` you want to change to `await`!